src/containers/search_settings.jsx

```diff
@@ -83,11 +86,48 @@ class SearchSettings extends Component {
       align-items: center;
     `;
 
+    const {
+      FacebookShareButton,
+      EmailShareButton,
+    } = ShareButtons;
+
+    const FacebookShareButtonSetting = {
+      icon: generateShareIcon('facebook'),
+      isRound: true,
+      title: 'GitHub',
+      shareUrl: 'http://github.com',
+      size: 32,
+    };
+
+    const EmailShareButtonSetting = {
+      icon: generateShareIcon('email'),
+      isRound: true,
+      body: 'shared from http://github.com',
+      size: 32,
+      title: 'Sharing SERLER Search Result',
+      shareUrl: 'http://github.com',
+    };
+
     return (
       <Container>
         <ModuleTitle>Search Results</ModuleTitle>
         <Content>
           {this.renderVisibility()}
+
+          <FacebookShareButton
+            url={FacebookShareButtonSetting.shareUrl}
+            quote={FacebookShareButtonSetting.title}
+          >
+            <FacebookShareButtonSetting.icon size={FacebookShareButtonSetting.size} round />
```

Comment:
If the settings just use for one time, it would be more clear to set the value directly other than assigning them to another variable.